packages/dashboard-core-plugins/src/panels/WidgetPanelTypes.ts

```diff
@@ -0,0 +1,61 @@
+import { ReactNode } from 'react';
+import { LayoutUtils } from '@deephaven/dashboard';
+import type { Container } from '@deephaven/golden-layout';
+
+export type WidgetPanelDescriptor = {
+  /** Type of the widget. */
+  type: string;
+
+  /** Name of the widget. */
+  name: string;
+
+  /** Display name of the widget. May be different than the assigned name. */
+  displayName?: string;
+
+  /** Description of the widget. */
+  description?: string;
+};
+
+/** @deprecated Use WidgetPanelTooltipPropsV1 instead */
+export type WidgetPanelTooltipPropsV1 = {
```

Comment:
V2 or just the non versioned and name this legacy or something?

```suggestion
/** @deprecated Use WidgetPanelTooltipPropsV2 instead */
export type WidgetPanelTooltipPropsV1 = {
```